2.05-custom-gx/proc.hsp

```diff
@@ -5108,6 +5108,14 @@
 	if ( synccheck(cc, -1) ) {
 		txt lang(npcn(cc) + itemname(ci, 1) + "を食べ終えた。", name(cc) + " " + have(cc) + " finished eating " + itemname(ci, 1) + ".")
 	}
+	if ( TweakData(TWEAK_GAMEPLAY_CG_DISABLE_THIRST, TWEAK_CATEGORY_GAMEPLAY) == FALSE ) {
```

Comment:
Doesn't this lack the check for cargo food? (`if ( refitem(inv(INV_ITEM_ID, ci), DBSPEC_CARGO) == 1 )`) It's fine to just uncomment the code below this that removes the mechanic instead.

Reply:
Lemme go un-comment that code real quick! And yes, you're 100% right that I forget the cargo food check(I tunnel visioned and only tested cargo food.) - this was before I learned `*eat` is a "gosub" that ALL foods go too. I'll just smack the tweak condition into the original and that should hopefully be a lot better. 
I just general avoid commented-out code since there's usually for a reason and it's best I don't touch what I don't understand.